src/main/java/strategy/handlerimpl/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package strategy.handlerimpl;
+
+import dao.StorageDao;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public Integer operate(FruitTransaction order, StorageDao inStock) {
+        return inStock.get(order.getFruit());
+    }
```

Comment:
have you reviewed and tested your solution? I suppose currently your program isn't working correctly. Let's make this operate() methods of your handlers to return null and perform all validation and calculation logic within them. As a result of operate() method call fruits in your storage should be updated. Also StorageDao should be passed to your handlers using a constructor